src/he/flow.c

```diff
@@ -67,19 +75,6 @@ he_return_code_t he_conn_inside_packet_received(he_conn_t *conn, uint8_t *packet
     return HE_ERR_FAILED;
   }
 
-  // Sanity-check length -- contract says that it can't be longer than length but just-in-case
-  if(post_plugin_length > length) {
```

Comment:
I don't think it can be longer than length because how is the plugin supposed to know the maximum size of the buffer?

Reply:
Good point, I didn't realize the last `length` argument was for "capacity". I'll revert the last commit.